query/src/test/java/org/infinispan/query/test/MemberType.java

```diff
@@ -0,0 +1,10 @@
+package org.infinispan.query.test;
+
+/**
+ * @since 9.1
+ */
+public enum MemberType {
```

Comment:
I will handle the tests with flag in a follow-up JIRA  https://issues.jboss.org/browse/ISPN-7828